.github/workflows/atfe_nightly_build_and_test.yml

```diff
@@ -29,27 +30,54 @@ jobs:
       fail-fast: false  # Prevents one job failure from cancelling all
       matrix:
         include:
+          # build.sh
```

Comment:
This comment doesn't add much context, as the script is mentioned directly below. Can we replace it with something like `Complete Toolchain Build & Test`?